packages/xod-client/src/debugger/messages.js

```diff
@@ -14,12 +14,19 @@ export const UPDATING_ARDUINO_DEPENDECIES = 'Updating Arduino packages...';
 export const UPDATING_ARDUINO_DEPENDECIES_SUCCESS =
   'All Arduino packages are updated successfully';
 
-export const TABTEST_GENERATING_CODE = 'Generating C++ code...';
-export const TABTEST_BUILDING = 'Building...';
-export const TABTEST_BUILDING_ERROR = `The generated C++ code contains errors. It can be due to a bad node implementation
+export const WASM_NO_RUNTIME_FOUND = 'Unable to get URL of WASM runtime script';
+export const WASM_BUILDING_ERROR = `The generated C++ code contains errors. It can be due to a bad node implementation
```

Comment:
Didn’t ESLint caught you? This line should be hard-wrapped

Reply:
Nope. But it does if I adds a new line before the string :)